Cargo.lock

```diff
@@ -2732,9 +2732,9 @@ checksum = "68354c5c6bd36d73ff3feceb05efa59b6acb7626617f4962be322a825e61f79a"
 
 [[package]]
 name = "miniz_oxide"
```

Comment:
Github seems to be broken, this is not a real file change, already on `0.8.6` on your branch @cody-wang-cb 
```
[[package]]
name = "miniz_oxide"
version = "0.8.6"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "29c72f6929239626840b28f919ce8981a317fc5dc63ce25c30d2ab372f94886f"
dependencies = [
 "adler2",
]
```